docs/user/translating.rst

```diff
@@ -1,21 +1,18 @@
 Translating using Weblate
 =========================
 
-Thank you for interest in translating using Weblate. Projects can either be
-set up for direct translation, or by way of accepting suggestions made by
-users without accounts.
+Projects either accept translations directly,
+or when a number of votes is reached.
```

Comment:
Suggestions can be used with voting, but it is not the only use case, thus it can’t be described like this.